test/Api.Tests/ControllerTests/UsersControllerTest.cs

```diff
@@ -1,70 +1,159 @@
-﻿using System.Security.Claims;
-using PaderbornUniversity.SILab.Hip.CmsApi.Controllers;
+﻿using PaderbornUniversity.SILab.Hip.CmsApi.Controllers;
 using PaderbornUniversity.SILab.Hip.CmsApi.Models;
 using PaderbornUniversity.SILab.Hip.CmsApi.Models.Entity;
 using MyTested.AspNetCore.Mvc;
 using Xunit;
+using System.Linq;
+using PaderbornUniversity.SILab.Hip.CmsApi.Utility;
+using PaderbornUniversity.SILab.Hip.CmsApi.Models.User;
 
 namespace PaderbornUniversity.SILab.Hip.CmsApi.Tests.ControllerTests
 {
     public class UsersControllerTest
     {
+        private ControllerTester<UsersController> _tester;
+        public InviteFormModel InviteFormModel { get; set; }
+
+        public UsersControllerTest()
+        {
+            _tester = new ControllerTester<UsersController>();
+            InviteFormModel = new InviteFormModel()
+            {
+                Emails = new string[]
+                        {
+                            "abc@xyz.com", "lmn@xyz.com"
+                        }
+            };
+        }
+
+        #region InviteUsers
+
+        /// <summary>
+        /// Should return code 202 when an admin is trying to invite new users with new Emails
+        /// </summary>
+        //[Fact] - Dependency Injection is not working with XUnit and the issue is still open
+        // Refer: https://github.com/xunit/xunit/issues/687
```

Comment:
Did you already try the alternative solution using a fixture class? I think that's the way to go here, as the issue you mentioned is closed.